app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
```

Comment:
If you are not using `@comment` variable in the view change it to `comment`
You can also do this without creating a variable by changing next line to
```
if @article.comment.last.persisted?
```

Reply:
made these changes